lib/commands/execute.js

```diff
@@ -26,14 +27,21 @@ extensions.executeAsync = async function (script, args, sessionId) {
 
 // Overrides the 'executeMobile' function defined in appium-ios-driver
 extensions.executeMobile = async function (mobileCommand, opts={}) {
-  // we only support mobile: scroll and mobile: swipe
-  if (mobileCommand === 'scroll') {
-    await this.mobileScroll(opts);
-  } else if (mobileCommand === 'swipe') {
-    await this.mobileScroll(opts, true);
-  } else {
-    throw new errors.UnknownCommandError('Unknown command, all the mobile commands except scroll and swipe have been removed.');
+  const mobileCommandsMapping = {
+    scroll: async (x) => {await this.mobileScroll(x);},
+    swipe: async (x) => {await this.mobileScroll(x, true);},
+    pinch: async (x) => {await this.mobilePinch(x);},
+    doubleTap: async (x) => {await this.mobileDoubleTap(x);},
+    twoFingerTap: async (x) => {await this.mobileTwoFingerTap(x);},
+    touchAndHold: async (x) => {await this.mobileTouchAndHold(x);},
+    tap: async (x) => {await this.mobileTap(x);},
+    dragFromToForDuration: async (x) => {await this.mobileDragFromToForDuration(x);}
+  };
+
+  if (!_.has(mobileCommandsMapping, mobileCommand)) {
+    throw new errors.UnknownCommandError(`Unknown mobile command "${mobileCommand}". Only ${_.keys(mobileCommandsMapping)} commands are supported.`);
   }
+  await mobileCommandsMapping[mobileCommand](opts);
```

Comment:
I know the current behavior also doesn't have `return` but it seems like that could be missing. however none of the proxy commands have `return` so we'd have to make this change everywhere if we cared about it.